pkg/brokerapi/openservicebroker/open_service_broker_client.go

```diff
@@ -138,7 +139,7 @@ func (c *openServiceBrokerClient) GetCatalog() (*brokerapi.Catalog, error) {
 	return &catalog, nil
 }
 
-func (c *openServiceBrokerClient) CreateServiceInstance(ID string, req *brokerapi.CreateServiceInstanceRequest) (*brokerapi.CreateServiceInstanceResponse, error) {
+func (c *openServiceBrokerClient) CreateServiceInstance(ID string, req *brokerapi.CreateServiceInstanceRequest) (*brokerapi.CreateServiceInstanceResponse, int, error) {
 	var serviceInstanceURL string
 
 	if req.AcceptsIncomplete {
```

Comment:
Actually, I was going to create another PR later on to remove it, I stumbled on this and was thinking of removing it, but did not want to change that behaviour yet. I wanted to spend little more time making sure I didn't miss anything in the spec on the significance here, but yes, I think we should remove it.